plugin/src/main/java/io/jenkins/plugins/coverage/metrics/model/Side.java

```diff
@@ -0,0 +1,6 @@
+package io.jenkins.plugins.coverage.metrics.model;
+
+public enum Side {
```

Comment:
I am not sure if this is really necessary for coverage since always the right side / new code is of interest.
I think this can be removed.